src/screens/signup/index.tsx

```diff
@@ -55,7 +64,23 @@ export default function SignupScreen(props: SignupScreenProp) {
             <Ionicons name="logo-facebook" size={30} color="#4267B2" />
           </IconView>
         </Button>
-      </SafeAreaView>
-    </Container>
+        <Button
+          title="Continue with Google"
+          buttonStyle={{
+            marginBottom: '6%',
+            height: 60,
+            borderColor: colors.GOOGLE_COLOR,
+            borderWidth: 1,
+            backgroundColor: colors.BG_WHITE_COLOR
+          }}
+          textStyle={{ color: colors.GOOGLE_COLOR }}
+        >
+          <IconView>
+            <GoogleIcon source={require('../../../assets/icons/google.png')} />
```

Comment:
can you change the google icon as a picture(.PNG) to use SVG so it can be cleaner and readable across all screen sizes even on tabs, just so we don't need to worry if the image is been to stretch and lose its quality, as SVG maintains pixel ratio on all screen sizes.